libraries/AP_HAL_ChibiOS/hwdef/OrqaF405Pro/hwdef.dat

```diff
@@ -0,0 +1,154 @@
+
+# hw definition file for processing by chibios_hwdef.py
+# for Orqa F405 PRO hardware.
+# thanks to betaflight for pin information
+
+# MCU class and specific type
+MCU STM32F4xx STM32F405xx
+
+# board ID for firmware load
+APJ_BOARD_ID 1155
+
+# crystal frequency, setup to use external oscillator
+OSCILLATOR_HZ 8000000
+
+FLASH_SIZE_KB 1024
+
+# bootloader takes first sector
+FLASH_RESERVE_START_KB 48
+
+define HAL_STORAGE_SIZE 16384
+define STORAGE_FLASH_PAGE 1
+
+STM32_ST_USE_TIMER 5
+
+# SPI devices
+
+# SPI1
+PA5 SPI1_SCK SPI1
+PA6 SPI1_MISO SPI1
+PA7 SPI1_MOSI SPI1
+
+# SPI2
+PB13 SPI2_SCK SPI2
+PB14 SPI2_MISO SPI2
+PB15 SPI2_MOSI SPI2
+
+# SPI3
+PB5 SPI3_MOSI SPI3
+PC10 SPI3_SCK SPI3
+PC11 SPI3_MISO SPI3
+
+# Chip select pins
+PB3 FLASH1_CS CS
+PB12 OSD1_CS CS
+PA4 GYRO1_CS CS
+
+# Beeper
+PA8 TIM1_CH1 TIM1 GPIO(80) ALARM
+PA8 BUZZER OUTPUT GPIO(81) LOW ALT(1)
```

Comment:
did not know alt config works on non UARTS!

Reply:
Seems to work